monero/wallet_client_test.go

```diff
@@ -6,29 +6,38 @@ import (
 	"path"
 	"testing"
 
+	"github.com/cockroachdb/apd/v3"
 	logging "github.com/ipfs/go-log"
 	"github.com/stretchr/testify/require"
 
+	"github.com/athanorlabs/atomic-swap/coins"
 	"github.com/athanorlabs/atomic-swap/common"
 	mcrypto "github.com/athanorlabs/atomic-swap/crypto/monero"
 )
 
 var moneroWalletRPCPath = path.Join("..", "monero-bin", "monero-wallet-rpc")
 
+// package "tests" depends on the "monero", so StrToDecimal is in "common" so both can use it.
```

Comment:
```suggestion
// package "tests" depends on the "monero" package, so StrToDecimal is in "common" so both can use it.
```

Reply:
Thanks for catching that. That was legacy code before I broke the coins off into their own package. I just updated the references here to use `coins.StrToDecimal` directly.